.github/workflows/test-plugins.yml

```
@@ -0,0 +1,31 @@
+name: Test Plugins
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+on:
+  pull_request:
+    paths:
+      - 'jekyll-kuma-plugins/**'
+      - 'app/_plugins/**'
+      - 'spec/**'
```

The workflow path filter is incomplete. Changes to `.rubocop.yml`, `mise.toml`, or this workflow file itself won't trigger the tests.

```suggestion
      - 'jekyll-kuma-plugins/**'
      - 'app/_plugins/**'
      - 'spec/**'
      - '.rubocop.yml'
      - '.github/workflows/test-plugins.yml'
      - 'mise.toml'
```